pkg/netpol/connlist/connlist_test.go

```diff
@@ -382,3 +458,12 @@ func TestWithFocusWorkloadWithReplicasConnections(t *testing.T) {
 	require.Nil(t, err)
 	require.NotContains(t, out, "kube-system/calico-node[DaemonSet] => kube-system/calico-node[DaemonSet] : All Connections")
 }
+
+func TestConnlistAnalyzerIllegalPodList(t *testing.T) {
+	dirPath := filepath.Join(testutils.GetTestsDir(), "semanticDiff-same-topologies-illegal-podlist")
+	analyzer := NewConnlistAnalyzer(WithIncludeJSONManifests())
+	res, _, err := analyzer.ConnlistFromDirPath(dirPath)
+	require.Nil(t, res)
+	require.Contains(t, err.Error(), "Input Pod resources are not supported for connectivity analysis."+
+		" Found Pods of the same owner but different set of labels.")
```

Comment:
Current error message output:
```
$ ./bin/k8snetpolicy list --dirpath tests/semanticDiff-same-topologies-illegal-podlist/ --include-json
Error: Input Pod resources are not supported for connectivity analysis. Found Pods of the same owner but different set of labels. Pod demo/account-query has label app with value account-query-selector, and Pod demo/bank-ui has label app with value bank-ui-selector

```
few more issues to address:
- Missing `.` at the end of the last sentence.
- Replace `has label (key) with value (val)` with `has label: key=val`
- Add details about the pods owner `Found pods of the same owner (namespace/owner-name)`